{
  try {
    return this.factory.call();
  }
 catch (  Exception e) {
    Throwables.propagate(e);
  }
  return null;
}
